File: examples/src/Slides/Layout/SetChildFooter.cs
using Aspose.slides.Export

/*
This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Slides for .NET API reference 
when the project is build. Please check https:#docs.nuget.org/consume/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.Slides for .NET API from http:#www.aspose.com/downloads, 
install it and then add its reference to this project. For any issues, questions or suggestions 
please feel free to contact us using http:#www.aspose.com/community/forums/default.aspx
*/

namespace Aspose.slides.Examples.CSharp.slides.Layout
{
    class SetChildFooter
    {
        public static void Run()
        {
            #ExStart:SetChildFooter
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Slides_Presentations_Layout()
            using (Presentation presentation = new Presentation(dataDir+"presentation.ppt"))
            {
                IMasterSlideHeaderFooterManager headerFooterManager = presentation.Masters[0].HeaderFooterManager
                headerFooterManager.SetFooterAndChildFootersVisibility(True) # Method SetFooterAndChildFootersVisibility is used for making a master slide and all child footer placeholders visible.
                headerFooterManager.SetSlideNumberAndChildSlideNumbersVisibility(True) # Method SetSlideNumberAndChildSlideNumbersVisibility is used for making a master slide and all child page number placeholders visible.
                headerFooterManager.SetDateTimeAndChildDateTimesVisibility(True) # Method SetDateTimeAndChildDateTimesVisibility is used for making a master slide and all child date-time placeholders visible.

                headerFooterManager.SetFooterAndChildFootersText("Footer text") # Method SetFooterAndChildFootersText is used for setting text to master slide and all child footer placeholders.
                headerFooterManager.SetDateTimeAndChildDateTimesText("Date and time text") # Method SetDateTimeAndChildDateTimesText is used for setting text to master slide and all child date-time placeholders.
            }

            #ExEnd:SetChildFooter
        }
    }
}

File: examples/src/Slides/Layout/SetChildFooter.py
import aspose.slides as slides


#ExStart:SetChildFooter
# The path to the documents directory.
dataDir = "./examples/data/"
outDir = "./examples/out/"
with slides.Presentation(dataDir + "layout_presentation.ppt") as presentation:
    headerFooterManager = presentation.masters[0].header_footer_manager
    headerFooterManager.set_footer_and_child_footers_visibility(True) # Method set_footer_and_child_footers_visibility is used for making a master slide and all child footer placeholders visible.
    headerFooterManager.set_slide_number_and_child_slide_numbers_visibility(True) # Method set_slide_number_and_child_slide_numbers_visibility is used for making a master slide and all child page number placeholders visible.
    headerFooterManager.set_date_time_and_child_date_times_visibility(True) # Method set_date_time_and_child_date_times_visibility is used for making a master slide and all child date-time placeholders visible.

    headerFooterManager.set_footer_and_child_footers_text("Footer text") # Method set_footer_and_child_footers_text is used for setting text to master slide and all child footer placeholders.
    headerFooterManager.set_date_time_and_child_date_times_text("Date and time text") # Method set_date_time_and_child_date_times_text is used for setting text to master slide and all child date-time placeholders.

#ExEnd:SetChildFooter